### PR TITLE
Add HealthDataStandards namespace to Procedure

### PR DESCRIPTION
**Pull request by v7chan**
Modified all the (similar) files identified by #19 and a few more that matched the pattern.

There were a couple files where Procedure is set/used in a switch statement.. unclear if namespacing will affect it and tests did not show any errors:
- Example 1: https://github.com/q-centrix/health-data-standards/blob/mongoid5/lib/health-data-standards/import/cda/communication_importer.rb#L45
- Example 2: https://github.com/q-centrix/health-data-standards/blob/mongoid5/lib/hqmf-parser/2.0/data_criteria_helpers/dc_definition_from_template_or_type_extract.rb#L172

In support of q-centrix/web#1641.

### CODE REVIEW

File: lib/health-data-standards/import/bundle/importer.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 require 'zip/zipfilesystem'
+
```
Comment (dillonwelch) [inline lib/health-data-standards/import/bundle/importer.rb]: Leave the spacing as-is to minimize the diff from upstream

File: test/unit/import/bundle/importer_test.rb [hunk 2]
```diff
@@ -1,6 +1,5 @@
 require 'test_helper'
 
-
```
Comment (dillonwelch) [inline test/unit/import/bundle/importer_test.rb:3]: Leave the spacing as-is to minimize the diff from upstream

**Comment by dillonwelch** [comment]
For example 1, I think you'll have to change it because of this line: https://github.com/q-centrix/health-data-standards/blob/mongoid5/lib/health-data-standards/models/reference.rb#L11

**Comment by dillonwelch** [comment]
I think you're OK for the second example. If I'm reading it correctly, that code is reading from the CAT1 files, which will have "Procedure" in them.

**Comment by v7chan** [comment]
Updated to fix the first example and the whitespace. Accidentally modified the newlines when I was trying to debug the zip decompression, d'oh.

**Comment by dillonwelch** [comment]
Can you update ecqm-api to bump the version of this lib too? Thanks!

**Pull request merged**